addons/website_event_booth_sale/models/sale_order.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import Command
+from odoo import models, _
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    def _cart_find_product_line(self, product_id=None, line_id=None, **kwargs):
+        self.ensure_one()
+        lines = super(SaleOrder, self)._cart_find_product_line(product_id, line_id)
+        if line_id:
+            return lines
+        event_booth_pending_ids = self.env.context.get('event_booth_pending_ids')
+        if event_booth_pending_ids:
```

Comment:
Could probably gain some clarity with a docstring explaining a bit what we do here :D